ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -5,4 +5,20 @@ import DynamicQuantities: UnionAbstractQuantity, ustrip, dimension, new_quantity
 
 norm(q::UnionAbstractQuantity, p::Real=2) = new_quantity(typeof(q), norm(ustrip(q), p), dimension(q))
 
+# Define isapprox for vectors of Quantity types
+function Base.isapprox(
+    u::AbstractArray{Q}, v::AbstractArray{Q};
+    atol=new_quantity(Q, zero(T), dimension(first(u))),
+    rtol=Base.rtoldefault(T),
+    nans::Bool=false, norm::Function=norm
+) where {T, D, Q<:UnionAbstractQuantity{T,D}}
+    d = norm(u - v)
+    if isfinite(d)
+        return d <= max(atol, rtol*max(norm(u), norm(v)))
+    else
+        # Fall back to a component-wise approximate comparison
+        return all(ab -> isapprox(uv[1], uv[2]; rtol=rtol, atol=atol, nans=nans), zip(u, v))
```

Comment:
Is this what normal isapprox does? Maybe add a test to assert the behavior is the same?

Reply:
The standard `Base.isapprox` method for vectors is defined by LinearAlgebra [here](https://github.com/JuliaLang/LinearAlgebra.jl/blob/8108bc4ff2dbd5b2143d6279efad640952f97f8e/src/generic.jl#L1768). This method for dimensionful types is pretty much a direct translation of that code but fixes `atol` and `rtol` handling.